src/module.ts

```diff
@@ -54,12 +57,28 @@ export default defineNuxtModule<ModuleOptions>({
       apiKey: options.apiKey
     }
 
-    addPlugin(resolver.resolve('./runtime/plugin'))
-    addImportsDir(resolver.resolve('./runtime/composables'))
+    addPlugin(resolve('./runtime/plugins/stripe.client'))
+    addImportsDir(resolve('./runtime/composables'))
 
     nuxt.hook('nitro:config', (nitroConfig) => {
       nitroConfig.alias = nitroConfig.alias || {}
-      nitroConfig.alias['#stripe'] = resolver.resolve('./runtime/services/index.ts')
+      nitroConfig.externals = defu(typeof nitroConfig.externals === 'object' ? nitroConfig.externals : {}, {
+        inline: [resolve('./runtime')]
+      })
+      nitroConfig.alias['#stripe/server'] = resolve('./runtime/server/services')
+    })
+
+    addTemplate({
```

Comment:
this will create a virtual file that will after being imports by nuxt with the prepare type hook